docs/API.md

```diff
@@ -18,6 +18,8 @@ import { AuthUIContextProvider } from '@pxblue/react-native-auth-workflow';
 
 #### Available Props
 
+-   **accountAlreadyExistsScreen**: (optional) _`JSX.Element | (navigation: any, registrationData?: RegistrationData) => JSX.Element`_
```

Comment:
Correct me if I am wrong, but I don't think we will have any `registrationData` to provide to the user in the 'account already exists' case.